samples/react-flow-dashboard/package.json

```diff
@@ -3,43 +3,46 @@
   "version": "0.0.1",
   "private": true,
   "engines": {
-    "node": ">=16.13.0 <17.0.0"
+    "node": ">=16.13.0 <17.0.0 || >=18.17.1 <19.0.0"
   },
   "main": "lib/index.js",
   "scripts": {
     "build": "gulp bundle",
     "clean": "gulp clean",
-    "test": "gulp test"
+    "test": "gulp test",
+    "serve": "gulp serve"
   },
   "dependencies": {
-    "@microsoft/sp-component-base": "1.17.4",
-    "@microsoft/sp-core-library": "1.17.4",
-    "@microsoft/sp-lodash-subset": "1.17.4",
-    "@microsoft/sp-office-ui-fabric-core": "1.17.4",
-    "@microsoft/sp-property-pane": "1.17.4",
-    "@microsoft/sp-webpart-base": "1.17.4",
+    "@fluentui/react": "8.106.4",
+    "@microsoft/sp-adaptive-card-extension-base": "1.20.0",
+    "@microsoft/sp-component-base": "1.20.0",
+    "@microsoft/sp-core-library": "1.20.0",
+    "@microsoft/sp-lodash-subset": "1.20.0",
+    "@microsoft/sp-office-ui-fabric-core": "1.20.0",
+    "@microsoft/sp-property-pane": "1.20.0",
+    "@microsoft/sp-webpart-base": "1.20.0",
     "@pnp/spfx-controls-react": "^3.15.0",
     "@pnp/spfx-property-controls": "^3.14.0",
     "jwt-decode": "^3.1.2",
-    "office-ui-fabric-react": "^7.199.1",
```

Comment:
seem this gets deleted and we do not replace it with anything else in code but actually it is imported here
https://github.com/pnp/sp-dev-fx-webparts/blob/5df1e8b4fc66c8a265ce5517d8cfb376489c9987/samples/react-flow-dashboard/src/webparts/reactFlowDashboard/components/Dashboard/IReactFlowDashboardState.ts#L1-L2

will it work? I guess it may not build/bundle